html2text_test.go

```diff
@@ -406,3 +446,52 @@ func assertPlaintext(t *testing.T, input string, matcher StringMatcher) {
 		t.Logf("input:\n\n%s\n\n\n\noutput:\n\n%s\n", input, text)
 	}
 }
+
+func ExampleSimpleUsage() {
```

Comment:
I've added this example to make it to appear in godoc as well, rather than in README file only.
See https://godoc.org/github.com/korya/html2text#example-package 

But I gave it a bad name and godoc didn't pick it up. Fixed it in next commit